source/DSCResources/DSC_CMSoftwareUpdatePoint/DSC_CMSoftwareUpdatePoint.psm1

```diff
@@ -0,0 +1,458 @@
+
+$script:localizedData = Get-LocalizedData -DefaultUICulture 'en-US'
+
+ <#
+    .SYNOPSIS
+        This will return a hashtable of results.
+
+    .PARAMETER SiteCode
+        Specifies the site code for Configuration Manager site.
+
+    .PARAMETER SiteServerName
+        Specifies the Site Server to install or configure the role on.
+
+    .Notes
+        This must be run on the Primary servers to install the software update role.
+        The Primary server computer account must be in the local
+        administrators group to perform the install.
+
+#>
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [String]
+        $SiteCode,
+
+        [Parameter(Mandatory = $true)]
+        [String]
+        $SiteServerName
+    )
+
+    Write-Verbose -Message $script:localizedData.RetrieveSettingValue
+    Import-ConfigMgrPowerShellModule -SiteCode $SiteCode
+    Set-Location -Path "$($SiteCode):\"
+
+    $supProps = (Get-CMSoftwareUpdatePoint -SiteCode $SiteCode -SiteSystemServerName $SiteServerName).Props
+
+    if ($supProps)
+    {
+        foreach ($supProp in $supProps)
+        {
+            switch ($supProp.PropertyName)
+            {
+                'AllowProxyTraffic' { $cloudGateway = $supProp.Value }
+                'IsINF'             {
+                                        if ($supProp.Value -eq '1')
+                                        {
+                                            $internet = 'Internet'
+                                        }
+                                    }
+                'IsIntranet'        {
+                                        if ($supProp.Value -eq '1')
+                                        {
+                                            $intranet = 'Intranet'
+                                        }
+                                    }
+                'SSLWSUS'           {$enableSsl = $supProp.Value}
+                'UseProxy'          {$useProxyGeneral = $supProp.Value}
+                'UseProxyForADR'    {$useProxyForADR = $supProp.Value}
+                'WSUSAccessAccount' {
+                                        $accessAccount = $supProp.Value2
+                                        if ([string]::IsNullOrEmpty($supProp.Value2))
+                                        {
+                                            $anonymousWsus = $true
+                                        }
+                                        else
+                                        {
+                                            $anonymousWsus = $false
+                                        }
+                                    }
+                'WSUSIISPort'       {$wsusIis = $supProp.Value}
+                'WSUSIISSSLPort'    {$wsusIisSsl = $supProp.Value}
+            }
+        }
+        if (($internet) -and ($intranet))
```

Comment:
shouldn't need all of the () 
this should work: 
if ($internet -and $intranet)

Reply:
Removed extra ()